packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -106,11 +107,16 @@ class RenderParagraph extends RenderBox
          strutStyle: strutStyle,
          textWidthBasis: textWidthBasis,
          textHeightBehavior: textHeightBehavior,
-       ) {
+       ),
+       _textLength = text.toPlainText(includeSemanticsLabels: false).length {
     addAll(children);
     _extractPlaceholderSpans(text);
   }
 
+  static final Paint _selectionPaint = Paint()
+    ..style = PaintingStyle.fill
+    ..color = const Color(0xAF6694e8);
```

Comment:
The color used by EditableText is more complicated than this, it's based on platform (e.g. [here](https://github.com/flutter/flutter/blob/e5f9d5bd26982922dca64acc9066e32fe806e12a/packages/flutter/lib/src/material/text_field.dart#L1170)).  Maybe we could pull that code out of TextField etc. and reuse it everywhere.

Reply:
ah right, we should add a theme for this